tests/MabeEnumTest/TestAsset/Enum31.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace MabeEnumTest\TestAsset;
+
+use MabeEnum\Enum;
+
+/**
+ * Unit tests for the class MabeEnum\Enum
+ *
+ * @link http://github.com/marc-mabe/php-enum for the canonical source repository
+ * @copyright Copyright (c) 2013 Marc Bennewitz
```

Comment:
It's 2017, maybe you should update that somewhen :)

Reply:
good point -> but on another branch